Makefile

```diff
@@ -1,7 +1,7 @@
 
 BIN ?= murmur
 SRC = murmurhash.c
-CFLAGS += -std=c99 -Wall -I.
+CFLAGS += -std=c99 -Wall -I. -I/usr/include -I/usr/include/sys -I/usr/include/machine
```

Comment:
are these needed?

Reply:
On MacOS /usr/include/endian.h reportedly is under /usr/include/machine
[ https://stackoverflow.com/questions/20813028/endian-h-not-found-on-mac-osx](url)